extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -161,8 +161,22 @@ export class AmpAnalytics extends AMP.BaseElement {
           if (!result) {
             return;
           }
-          addListener(this.getWin(), trigger,
-              this.handleEvent_.bind(this, trigger));
+
+          if (trigger['selector']) {
+            // Expand the selector using variable expansion.
+            trigger['selector'] = this.expandTemplate_(trigger['selector'],
+                trigger);
+            return urlReplacementsFor(this.getWin()).expand(trigger['selector'])
```

Comment:
For the usecases I am aware of and the ones described in this issue, I think going with this.expandTemplate_ is enough. In that case, only the vars that publishers specify will get expanded and not the ones that platform specifies. 

The reason to add  urlReplacementsFor is of course: uniformity across all expansions.

One solution could be to remove urlReplacementsFor for now and add it back if someone wants it.